server/helpers/verifyUser.js

```diff
@@ -25,5 +27,53 @@ class VerifyUser {
     }
     return done('password or email is incorrect');
   }
+
+
+  /**
+     * Retrieve hashed the password.
```

Comment:
Update this comment. **the** should come before hashed.

Reply:
Feedback Implemented!